examples/dishwasher-app/silabs/src/ElectricalSensorManager.cpp

```diff
@@ -0,0 +1,251 @@
+    int64_t ReactivePower;   // mW
+    int64_t ApparentPower;   // mW
+    int64_t RMSVoltage;      // mV
+    int64_t RMSCurrent;      // mA
+    int64_t RMSPower;        // mW
+    int64_t Frequency;       // Hz
+    int64_t PowerFactor;     // Percent 100ths
+    int64_t NeutralCurrent;  // mA
+} kAttributes[4] = {
+    { PowerModeEnum::kAc, 120'000, 0, 0, 0, 0, 0, 0, 120'000, 0, 0, 60, 98'00, 0 }, // kStopped
+    { PowerModeEnum::kAc, 120'000, 15'000, 17'000, 23'000, 1800'000, 2040'000, 3000'000, 120'000, 15'000, 1800'000, 60, 92'00,
+      15'000 },                                                                                                   // kRunning
+    { PowerModeEnum::kAc, 120'000, 125, 150, 190, 17'000, 18'000, 25'000, 120'000, 125, 17'000, 60, 95'00, 125 }, // kPaused
+    { PowerModeEnum::kUnknown, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }                                            // kError
+};
+
+/*
+ *  @brief  Creates a Delegate and Instance for Electrical Power/Energy Measurement and Power Topology clusters
+ *
+ * The Instance is a container around the Delegate, so
+ * create the Delegate first, then wrap it in the Instance
+ * Then call the Instance->Init() to register the attribute and command handlers
+ */
+CHIP_ERROR ElectricalSensorManager::Init()
+{
+    EndpointId EPMEndpointId = DataModelHelper::GetEndpointIdFromCluster(ElectricalPowerMeasurement::Id);
+    EndpointId EEMEndpointId = DataModelHelper::GetEndpointIdFromCluster(ElectricalEnergyMeasurement::Id);
+    EndpointId PTEndpointId  = DataModelHelper::GetEndpointIdFromCluster(PowerTopology::Id);
+    CHIP_ERROR err;
+
+    VerifyOrReturnError((EPMEndpointId != kInvalidEndpointId) && (EEMEndpointId != kInvalidEndpointId) &&
+                            (PTEndpointId != kInvalidEndpointId),
+                        CHIP_ERROR_INCORRECT_STATE, ChipLogError(AppServer, "EPM, EEM or PT Cluster not configured"));
+
+    // Initialize EPM (Electrical Power Management)
+    VerifyOrReturnError(!gEPMDelegate && !gEPMInstance, CHIP_ERROR_INCORRECT_STATE,
+                        ChipLogError(AppServer, "EPM Delegate or Instance already exist"));
+
+    gEPMDelegate = std::make_unique<ElectricalPowerMeasurementDelegate>();
+    VerifyOrReturnError(gEPMDelegate, CHIP_ERROR_NO_MEMORY, ChipLogError(AppServer, "Failed to allocate memory for EPM Delegate"));
+
+    /* Manufacturer may optionally not support all features, commands & attributes */
+    /* Turning on all optional features and attributes for test certification purposes */
+    gEPMInstance = std::make_unique<ElectricalPowerMeasurementInstance>(
+        EPMEndpointId, *gEPMDelegate,
+        BitMask<ElectricalPowerMeasurement::Feature, uint32_t>(
+            ElectricalPowerMeasurement::Feature::kDirectCurrent, ElectricalPowerMeasurement::Feature::kAlternatingCurrent,
+            ElectricalPowerMeasurement::Feature::kPolyphasePower, ElectricalPowerMeasurement::Feature::kHarmonics,
```

Comment:
Would you have a PolyPhase Dishwasher/Washing machine in the real world and Harmonics and PowerQuality - seems overkill to me?